parser/src/sourcemanager.cpp

```diff
@@ -221,6 +234,30 @@ void SourceManager::updateFile(const model::File& file_)
     });
 }
 
+void SourceManager::removeFile(const model::File& file_)
+{
+  bool removeContent = false;
+
+  // Delete File and FileContent (only when no other File references it)
+  _transaction([&, this]() {
```

Comment:
Why is `this` captured here, if everything also captured by reference?

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.

Reply:
Fixed in 52b7b72b31bdc585c43440a29071f0842e4570b6.